### ISSUE TITLE
Visual label is not provided for "filter network" search box under Networks Tab panel.: A11y_Playwright OSS releases_Labels or Instructions

### ISSUE THREAD

**Issue by ganagdhar123** [microsoft/playwright.dev#1802]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### GitHub Tags:
#A11yTCS;#Playwright OSS Releases;#BM_PlaywrightOSSReleases_External_Sep2025;#ChromiumEdge;#DesktopWeb;#FTP;#Win11;#A11ySev2;#A11yMAS;#WCAG3.3.2;
### Environmental Details:
App Name: Playwright OSS releases
#URL: https://playwright.dev/
OS & Browser Details:
Microsoft Windows 11 Enterprise
Version 26100.3323
Microsoft Edge
Version 131.0.2903.99 (Official build) (64-bit)
### Repro Steps:

1. Hit the URL: https://playwright.microsoft.com/workspaces/eastus_cdbdcaba-d6e1-47a0-80b2-e586bb4a7247 and sign in with workspace account.
2. "Home" page will appear. TAB to any "Test run" and hit ENTER key.
3. TAB to any test case and hit ENTER key. TAB to "View trace viewer" button and hit ENTER key.
4. TAB to "Networks" tab and press enter.
5. Tab till "Filter network" search box.
6. Observe that Visual label is provided for "filter network" search box under Networks Tab panel or not.

### Actual:
Visual label is not provided for "filter network" search box under Networks Tab panel.
### Expected:
Visual label should be provided for "filter network" search box under Networks Tab panel.
### User Impact:
Cognitive users will face difficulty to understands the purpose and context of the control as there is no instruction and visual label is provided for the control.
### Attachment:

![Image](https://github.com/user-attachments/assets/fc9bb460-778c-4205-b4bd-2264dd15ed47)
